x-pack/plugins/ml/public/application/components/job_selector/job_selector.tsx

```diff
@@ -183,6 +185,15 @@ export function JobSelector({ dateFormatTz, singleSelection, timeseriesOnly }: J
               })}
             </EuiButtonEmpty>
           </EuiFlexItem>
+
+          <EuiFlexItem />
+
+          <EuiFlexItem grow={false}>
+            <FeedBackButton
```

Comment:
If you open the job selector and don't make any changes to the selected jobs, the feedback button isn't rendered. It oly renders for me when I change the selection.

Reply:
Fixed by https://github.com/elastic/kibana/pull/175613/commits/d2653e400be7a8936aa8da70e77bcaf0aa8bb6a4